samples/web/content/apprtc/js/main.js

```diff
@@ -139,7 +139,12 @@ function disconnectFromRoom() {
   // When the other client sees BYE it attempts to post offer and candidates to
   // GAE. GAE needs to know that we're disconnected at that point otherwise
   // it will forward messages to this client instead of storing them.
-  path = '/bye/' + params.roomId + '/' + params.clientId;
+  var server = params.server;
+  if (!server) {
+    // If server is not provided, use relative URI.
+    server = '';
+  }
```

Comment:
This is duplicated in a few places. Why not set params.server at the beginning of initialize()?